Serenity-Website/src/App.jsx

```diff
@@ -5,12 +5,15 @@ import LandingPage from './Landing-Page/LandingPage.jsx';
 import SignUp from './Sign-up/SignUp.jsx';
 import MoodBoard from './MoodBoard/MoodBoard.jsx';
 import LogIn from './LogIn/LogIn.jsx';
+import { UserContext } from '../../UserContext.js';
+import Preloader from './Preloader/Preloader.jsx';
 
 function App() {
   const [user, setUser] = useState(() =>{
+    
     //Retrieve user data from storage or set to null if not found
     const storedUser = localStorage.getItem('user');
-    return storedUser ? JSON.parse(storedUser) : null;
+    // return storedUser ? JSON.parse(storedUser) : null;
```

Comment:
Delete all commented out code that aren't explanatory.

Reply:
✅